.github/workflows/code_checks.yml

```diff
@@ -27,79 +22,108 @@ jobs:
       - name: Run Brakeman
         run:  bundle exec brakeman --ensure-latest --confidence-level=2 -o brakeman.json
 
-      - uses: duderman/rubocop-annotate-action@v0.1.0
+      - uses: actions/upload-artifact@v2
+        if: always()
         with:
-          path: rubocop.json
-        if: hashFiles('rubocop.json') != ''
-
-      - uses: devmasx/brakeman-linter-action@v1.0.0
-        env:
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-          REPORT_PATH: brakeman.json
-        if: hashFiles('brakeman.json') != ''
+          name: Linting and Security Results
+          path: |
+            rubocop.json
+            brakeman.json
+          if-no-files-found: ignore
 
   tests:
     name: Test
-    runs-on: asg
     env:
       BUNDLE_ENTERPRISE__CONTRIBSYS__COM: ${{ secrets.BUNDLE_ENTERPRISE__CONTRIBSYS__COM }}
       CI: true
       DATABASE_URL: "postgis://vets-api:postgres@127.0.0.1:5432/vets_api_test?pool=4"
-      DISABLE_BOOTSNAP: true
+      NODE_EXTRA_CA_CERTS: '/etc/ssl/certs/VA-Internal-S2-RCA1-v1.cer.pem' # Needed for self-hosted runners (setup-ruby)
       RAILS_ENV: test
-      "Settings.test_database_url": "postgis://vets-api:postgres@127.0.0.1:5432/vets_api_test"
+      "Settings.binaries.clamdscan": "clamscan"
       "Settings.saml.cert_path": "spec/support/certificates/ruby-saml.crt"
       "Settings.saml.key_path": "spec/support/certificates/ruby-saml.key"
-      "Settings.binaries.clamdscan": "clamscan"
+      "Settings.test_database_url": "postgis://vets-api:postgres@127.0.0.1:5432/vets_api_test"
       TERM: xterm-256color
-      GIT_COMMIT_SHA: ${{ github.sha }}
-      CC_TEST_REPORTER_ID: ${{secrets.CC_TEST_REPORTER_ID}}
-      ImageOS: ubuntu20
-      NODE_EXTRA_CA_CERTS: '/etc/ssl/certs/VA-Internal-S2-RCA1-v1.cer.pem' # Needed for self-hosted runners
+    runs-on: asg
 
     services:
       postgres:
         image: postgis/postgis:11-2.5
         env:
-          POSTGRES_USER: vets-api
           POSTGRES_DB: vets-api_test
```

Comment:
Should we rename this `vets_api_test` so it's consistent with  line 45?

Reply:
Turns out this line does nothing.